README.md

```diff
@@ -1,2 +1,21 @@
-# rcl
```

Comment:
I've been adding separate README.md to each package, so the repository level readme doesn't get overwhelmed with boilerplate.

Reply:
Got it! For now I'm adding with this PR just the part of the Readme needed for QD of rcl